src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany
```

Comment:
Relation beetwen movies and actors are many-to-many
```suggestion
    @ManyToMany
    @JoinTable(name = "movies_actors",
                    joinColumns = @JoinColumn(name = "movie_id"),
                    inverseJoinColumns = @JoinColumn(name = "actor_id"))
```